src/Auth/Startup.cs

```diff
@@ -91,6 +91,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseOpenIddict();
 
+            app.UseOpenIdConnectServer(configuration =>
```

Comment:
We were planning to run our application in docker containers with 'http' environment. We ran into an error "OpenIdConnectServer is not configured to allow insecure http". Though we already set it in ` services.AddOpenIddict<ApplicationUser, ApplicationDbContext>()
                .Configure(options =>
                {
                    options.AllowInsecureHttp = true;
})`

So,
`app.UseOpenIdConnectServer(configuration =>
            {
                configuration.AllowInsecureHttp = true;                
});`
We had to add this as a "Temporary fix" for us to work, although it doesn't make any sense even for us.